test10.md

```diff
@@ -0,0 +1,55 @@
+# Test task 10
+
+## Join our chat: [![Gitter](https://badges.gitter.im/Kottans/frontend.svg)](https://gitter.im/Kottans/frontend?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+[![MIT Licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Kottans/web/blob/master/LICENSE.md)
+[![Ideas and useful links](https://img.shields.io/badge/google--doc-ideas-ff69b4.svg)](https://docs.google.com/spreadsheets/d/1bZJhYjK3VHOS2HmQb2Fs4aHfEBt8mp1F09j9nEEDaqE/edit#gid=818017811)
+[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome#front-end-development)
+[![ghit.me](https://ghit.me/badge.svg?repo=Kottans/frontend)](https://ghit.me/repo/Kottans/frontend)
+
+When you're done with your challenge push your code to github repo and publish
+it using gh-pages. _Add link to it to your main frontend tasks repo._
+
+If you honestly finished all the previous steps than go ahead and share it with
+others - send a message in gitter channel with the link to you repo and "@/all"
+tag and the words "Weather APP - finished" and add link to repo with code of the
+website.
+
+## Weather app
+
+Create a page with a weather cast.
+
+As a user I would like to:
+
+* enter name of the city or it's postcode and get weather cast in this city
+* choose period of the weather cast: today, tomorrow, week, two weeks
+* choose units — Celsius or Fahrenheit
+* have a list of favorite cities. By selecting favorite city from the list, I
+  want to see weather cast in it
+* have a list of recently viewed cities
+* see current city in the link in the address bar of the browser. If I open this
+  link in the new window, website must show me weather in this city
+* see fancy icons for weather cast and design. Using of template absolutely

```

Comment:
for some reason I feel like 
`Using of template` is somewhat strange. Can `Using template` or `Usage of template`  be a better fit, maybe?